package.json

```diff
@@ -106,6 +108,7 @@
     "vue-template-compiler": "^2.4.2",
     "webpack": "^3.5.2",
     "webpack-dev-server": "^2.7.1",
+    "webpack-env": "^0.8.0",
```

Comment:
What is this needed for?

Reply:
Oh sorry, it was for testing, not useful here.